BookingsApi/BookingsApi.AcceptanceTests/BookingsApi.AcceptanceTests.csproj

```diff
@@ -29,7 +29,7 @@
     <PackageReference Include="SpecFlow" Version="3.9.40" />
     <PackageReference Include="SpecFlow.NUnit" Version="3.9.40" />
     <PackageReference Include="SpecFlow.Tools.MsBuild.Generation" Version="3.9.40" />
-    <PackageReference Include="VH.AcceptanceTests.Common" Version="1.21.64" />
+    <PackageReference Include="VH.AcceptanceTests.Common" Version="1.21.68" />
```

Comment:
Will this break the current pipeline as this version doesn't exist within the current Artiface feed?